shared/fs/container.js

```diff
@@ -0,0 +1,16 @@
+// @flow
+import {connect, type TypedState, type Dispatch} from '../util/container'
+import * as FSGen from '../actions/fs-gen'
+import Fs from '.'
+
+const mapStateToProps = (state: TypedState) => ({
+  counter: state.fs.counter,
+  you: state.config.username,
+})
+
+const mapDispatchToProps = (dispatch: Dispatch) => ({
+  increase: () => dispatch(FSGen.createIncreaseCount({amount: 1})),
```

Comment:
Does `dispatch(...)` here return an object to be merged later? How is `dispatch` implemented? Also, is `increase` and `increase10` like event names? I see they are functions on the component props. Does redux map them automatically?

Reply:
Dispatch comes from redux itself. How it's handled is dependent on how you configure redux. You should watch the video I mention in the ticket